### ISSUE TITLE
Axis ranges rendered wrong when using the axis header + setting the left axes container layout to vertical

### ISSUE THREAD

**Issue by hesselmeulenbeld** [amcharts/amcharts5#891]
When using the yAxis header container to add a label:

```
yAxis.axisHeader.children.push(
  am5.Label.new(root, {
    html: "<span style='font-size: 20px'>Axis header title</span>"
  })
);
```
and set the left axis container layout to vertical:

```
chart.leftAxesContainer.set('layout', root.verticalLayout);
```

The range lines are rendered in the wrong location. See:
https://codepen.io/hesselmeulenbeld/pen/KKGgRgb

**Comment by zeroin** [comment]
Well, this is expected behavior. I would recommend adding title to some other container, like topAxesContainer.

**Comment by hesselmeulenbeld** [comment]
Well, it still is weird, all other data is drawn correctly relative to the y-axis (even the range labels). Only the range lines are drawn in the wrong location. When using above=false for the range item, the range is drawn in the correct location. So it only goes wrong with above=true.

**Comment by martynasma** [comment]
Fixed in [5.3.12](https://github.com/amcharts/amcharts5/releases/tag/5.3.12).

## [5.3.12] - 2023-05-05

### Added
- New locale: Faroese (fo_FO). Thanks Martin Puge.

### Changed
- The `min` and `max` settings are now optional for `IHeatRule` ([Issue 883](https://github.com/amcharts/amcharts5/issues/883)).
- Removed `xlsx` package (with vulnerability) from dependencies in favor of a bundled hard copy (version 0.19.3) due to their decision not to publish to NPM. ([Issue 897](https://github.com/amcharts/amcharts5/issues/897)).

### Fixed
- Events now work properly with open shadow DOM (but not closed shadow DOM).
- Fixed export of data freezing in some cases where there are visual elements on the chart that are constantly being updated.
- Using parentheses in the in-line number formatting function was not working as needed.
- Tapping on a touch display outside chart was applying default state to elements that weren't being "touch-hovered".
- `getDataItemById()` method was not working properly on `Hierarchy` series.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
